pragma/tests/client_test.py

```diff
@@ -331,3 +322,15 @@ async def test_client_oracle_mixin_future(pragma_client: PragmaClient, contracts
     assert res.num_sources_aggregated == 2
     assert res.last_updated_timestamp == timestamp + 10
     assert res.decimals == 8
+
+
+def test_client_with_http_network():
+    with aioresponses() as mock:
+        mock.post(
+            url="http://test.rpc/rpc",
+            payload={"id": 1, "jsonrpc": "2.0", "result": "0x534e5f474f45524c49"},
+        )
+        client = PragmaClient(
+            network="http://test.rpc/rpc",
+        )
+        assert client.network == "testnet"
```

Comment:
A simple unit test to ensure there's no regression. `aioresponses` is used since the `FullNodeClient` uses async calls even though the sync version of the method is called (discovered through using debugger).